dlp/snippets/inspect/inspect_bigquery.go

```diff
@@ -193,7 +193,7 @@ func inspectBigquery(w io.Writer, projectID string, infoTypeNames []string, cust
 		}
 	})
 	if err != nil {
-		return fmt.Errorf("Receive: %w", err)
+		return fmt.Errorf("Receive: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.